### DISCUSSION TITLE
Error al cargar la primera línea

### DISCUSSION THREAD

**Discussion by Antiago1** [rramosp/ai4eng.v1#266] [01 - INTRODUCCIÓN]
He intentado abordar los dos primeros ejercicios, pero me he encontrado con un inconveniente. Al ejecutar la primera línea, he notado que el tiempo de espera es excesivamente largo o, la línea simplemente no se ejecuta. He probado distintos navegadores y, lamentablemente, el problema persiste.

Curiosamente, he tenido éxito en los otros módulos sin enfrentar problemas similares.  

Si alguien ha experimentado algo similar o tiene alguna idea sobre cómo solucionar este inconveniente, estaría muy agradecido por cualquier sugerencia que puedan ofrecer.

![image](https://github.com/rramosp/ai4eng.v1/assets/142853038/d18b5a9c-980d-40a0-a6a1-595dc225cc16)

**Comment by Felipe-RA** [comment]
Saludos @Antiago1 

No puedo replicar tu problema desde distintas máquinas y cuentas de google.

Si es posible, pasate por alguna de las sesiones sincrónicas y lo miramos con calma!

**Reply by Antiago1** [reply]
Gracias @Felipe-RA, 
El día de hoy ha estado funcionando bien, gracias por tu pronta respuesta
